.gitignore

```diff
@@ -57,3 +56,9 @@ bin/
 
 # trained models
 **/models/*
+
+# # logs
+**/logs/*
+
+
+!.gitkeep
```

Comment:
is this really needed? .gitkeep file should be kept by default

Reply:
I had issues on my machine that the folder would not show up without this. But could be possible to remove it. Any idea what the harm could be leaving it in?

Reply:
I do not think it harms anything, at most just double insurance for that files not be ignored at all.

Reply:
I'll keep it then